plugins/module_utils/redfish_utils.py

```diff
@@ -2409,7 +2409,6 @@ def set_boot_order(self, boot_list):
             return {"ret": False, "msg": "Key BootOrder not found"}
 
         boot = data["Boot"]
-        boot_order = boot["BootOrder"]
```

Comment:
Maybe this should stay commented, with a TODO tag: it is going to be needed to implement idempotency for this BootOrder setting.

Reply:
553ead50b8b781f9b2dfa329f46fbdb3dafbd33b